harness/determined/deploy/aws/gen_vcpu_mapping.py

```diff
@@ -11,7 +11,17 @@
 def _fetch_vcpu_mapping() -> Iterable[Tuple[str, Dict]]:
     # Price List api is only available in us-east-1 and ap-southeast-1.
     client = boto3.client("pricing", region_name="us-east-1")
-    for page in client.get_paginator("get_products").paginate(ServiceCode="AmazonEC2"):
+    for page in client.get_paginator("get_products").paginate(
+        ServiceCode="AmazonEC2",
+        MaxResults=100,
+        Filters=[
+            {
+                "Field": "regionCode",
+                "Type": "TERM_MATCH",
+                "Value": "us-east-1",
```

Comment:
The comment about supported regions is technically wrong now: https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/price-changes.html

But picking one region presumably deduplicates the results, right? Otherwise you'd get price results for the same resource in all supported regions?

Reply:
right. we haven't run this script forever. I tried running it as is, and gave up on waiting after 30 minutes. AWS just has too much stuff.